docs/content/preview/migrate/yb-voyager/yb-voyager-cli.md

```diff
@@ -0,0 +1,146 @@
+---
+title: YugabyteDB Voyager CLI
+linkTitle: YugabyteDB Voyager CLI
+description: YugabyteDB Voyager CLI and SSL connectivity.
+beta: /preview/faq/general/#what-is-the-definition-of-the-beta-feature-tag
+menu:
+  preview:
+    identifier: yb-voyager-cli
+    parent: yb-voyager
+    weight: 105
+isTocNested: true
+showAsideToc: true
+---
+
+yb-voyager is a command line executable program that supports migrating databases from PostgreSQL, Oracle, and MySQL to a YugabyteDB database.
+
+## Syntax
+
+```sh
+yb_voyager [ <migration-phase>... ] [ <arguments> ... ]
+```
+
+- *migration-phase*: See [Migration phase](#migration-phase)
+- *arguments*: See [Arguments](#arguments)
+
+### Command line help
+
+To display the available online help, run:
+
+```sh
+yb_voyager --help
+```
+
+### Migration phase
+
+The following command line options specify the migration phases.
```

Comment:
I sort of feel like eventually this should be a proper command reference, with the syntax and all options etc, rather than just links to the instructions.